admin/InitDb.pl

```diff
@@ -228,14 +228,9 @@ sub Create
     $system_sql->auto_commit;
     $system_sql->do("ALTER DATABASE $dbname SET timezone TO 'UTC'");
 
-    # You can do this via CREATE FUNCTION, CREATE LANGUAGE; but using
-    # "createlang" is simpler :-)
-    my $sys_db = Databases->get($sysname);
-    my $sys_in_thisdb = $sys_db->meta->clone_object($sys_db, database => $dbname);
-    my @opts = $sys_in_thisdb->shell_args;
-    splice(@opts, -1, 0, "-d");
-    $ENV{"PGPASSWORD"} = $sys_db->password;
-    system "createlang", @opts, "plpgsql";
+    $system_sql->auto_commit;
+    $system_sql->do("CREATE OR REPLACE LANGUAGE plpgsql");
```

Comment:
Quoting https://www.postgresql.org/docs/9.5/static/plpgsql-overview.html

```
In PostgreSQL 9.0 and later, PL/pgSQL is installed by default.
```

So I guess the answer is "no."